### PR TITLE
Disable --coverfile flag in Syzkaller for Cuttlefish

### PR DESCRIPTION
**Pull request by Huiziyang**
Syzkaller doesn't support --coverfile flag anymore, disable for Cuttlefish because physical devices are still using the old Syzkaller binaries.

### CODE REVIEW

File: src/python/bot/fuzzers/syzkaller/engine.py [hunk 1]
```diff
@@ -151,7 +151,8 @@ def fuzz(self, target_path, options, unused_reproducers_dir=None, max_time=0):
     self._create_temp_corpus_dir('new')
 
     args = options.arguments
-    args += ['--coverfile', runner.get_cover_file_path()]
+    if not environment.is_android_cuttlefish():
+      args += ['--coverfile', runner.get_cover_file_path()]
```
Comment (oliverchang) [inline src/python/bot/fuzzers/syzkaller/engine.py:155]: Add TODO here to remove this and implement something to call the HTTP endpoint ourselves?

**Review by oliverchang — APPROVED** [review]

File: src/python/bot/fuzzers/syzkaller/engine.py (hunk 1 above)
Comment (Huiziyang) [reply src/python/bot/fuzzers/syzkaller/engine.py:157]: Done

**Pull request merged**